myapp/src/Day2/Que17/CopyButton.jsx

```diff
@@ -0,0 +1,18 @@
+import React,{useState} from 'react';
+import useClipboard from './useClipboard'; // Import your custom hook
+
+function CopyButton() {
+  const { copiedText, copyToClipboard } = useClipboard();
+  const [text,setText] =useState("")
+
+  return (
+    <div>
+      <input type="text" onChange={(e)=> setText(e.target.value)} />
+      <button onClick={() => copyToClipboard(text)}>
+        {copiedText ? 'Copied!' : 'Copy to Clipboard'}
```

Comment:
@anantmishra-successive The button text is static. If I type a new word, it still shows "Copied"